lib/genesis/helpers.go

```diff
@@ -357,34 +358,40 @@ func addAuthoritiesValues(k1, k2 string, kt crypto.KeyType, value []byte, gen *G
 
 	// decode authorities values into []interface that will be decoded into json array
 	ava := [][]interface{}{}
-	buf := &bytes.Buffer{}
-	sd := scale.Decoder{Reader: buf}
-	_, err := buf.Write(value)
+	reader := new(bytes.Buffer)
+	_, err := reader.Write(value)
 	if err != nil {
 		return err
 	}
 
-	alen, err := sd.DecodeInteger()
+	var alen int
+	err = scale.Unmarshal(value, &alen)
 	if err != nil {
 		return err
 	}
-	for i := 0; i < int(alen); i++ {
+	for i := 0; i < alen; i++ {
 		auth := []interface{}{}
 		buf := make([]byte, 32)
-		if _, err = sd.Reader.Read(buf); err == nil {
+		if _, err = reader.Read(buf); err == nil {
 			var arr = [32]byte{}
 			copy(arr[:], buf)
-			pa, err := bytesToAddress(kt, arr[:])
-			if err != nil {
-				return err
+			pa, e := bytesToAddress(kt, arr[:])
```

Comment:
I think we could keep `err` as variable name here

Reply:
agreed. I was getting linter issues which is why I changed it but im gonna go fix those to be consistent